Do not install cclib tests and data files together with the Python module.
--- a/setup.py
+++ b/setup.py
@@ -37,24 +37,7 @@
 
     # Setup the list of packages.
     cclib_packages = ['cclib',
-                      'cclib.parser', 'cclib.progress', 'cclib.method', 'cclib.bridge',
-                      'cclib.test']
-
-    # Setup the list of data files.
-    cclib_prefix = 'lib/python%i.%i/site-packages/cclib' %(sys.version_info[0], sys.version_info[1])
-    test_prefix = cclib_prefix + '/test'
-    data_prefix = cclib_prefix + '/data'
-    cclib_datafiles = [ (cclib_prefix, ['ANNOUNCE', 'CHANGELOG', 'INSTALL', 'LICENSE', 'README', 'THANKS']),
-                        (test_prefix, ['test/testdata']),
-                        (data_prefix, ['data/regressionfiles.txt', 'data/wget.sh'])]
-    for program in programs:
-        data_dirs = os.listdir('data/%s' %program)
-        for data_dir in data_dirs:
-            if data_dir[:5] == 'basic':
-                dest = '%s/%s/%s' %(data_prefix, program, data_dir)
-                path = 'data/%s/%s' %(program, data_dir)
-                newfiles = ['%s/%s' %(path,fname) for fname in os.listdir(path) if fname[0] != '.']
-                cclib_datafiles.append((dest, newfiles))
+                      'cclib.parser', 'cclib.progress', 'cclib.method', 'cclib.bridge' ]
 
     setup(
         name = "cclib",
@@ -69,10 +52,8 @@
         long_description = "\n".join(doclines[2:]),      
         classifiers = filter(None, classifiers.split("\n")),
         platforms = ["Any."],
-        scripts = ["src/scripts/ccget", "src/scripts/cda"],
-        package_dir = {'cclib':'src/cclib', 'cclib.test':'test'},
-        packages = cclib_packages,
-        data_files = cclib_datafiles )
+        package_dir = {'cclib':'src/cclib'},
+        packages = cclib_packages )
 
 if __name__ == '__main__':
     setup_cclib()
